mco_ovs_supervisor.yml.tmpl

```diff
@@ -10,20 +10,24 @@ spec:
       version: 3.1.0
     systemd:
       units:
+        - contents:
+          name: setup-ovs.service
+          enabled: false
         - contents: |
             [Unit]
-            After=NetworkManager.service openvswitch.service
+            Before=kubelet.service
+            After=NetworkManager.service
             [Service]
             Type=oneshot
-            ExecStart=/bin/sh /var/setup-ovs.sh
+            ExecStart=/bin/sh /var/init-interfaces.sh
             [Install]
             WantedBy=multi-user.target
-          name: setup-ovs.service
+          name: init-interfaces.service
           enabled: true
     storage:
       files:
         - contents:
             source: data:text/plain;charset=utf-8;base64,${base64_script_content}
           filesystem: root
           mode: 484
-          path: /var/setup-ovs.sh
+          path: /var/init-interfaces.sh
```

Comment:
You haven't updated this on the _workers file

Reply:
oh good catch. thanks!